.github/workflows/main.yml

```diff
@@ -130,5 +130,54 @@ jobs:
         with:
          name: app reports
          path: app/build/test-results
-      
-      
+
+  bazel_tests:
+    name: Bazel tests
```

Comment:
More like: ``bazel_build_app`` since we aren't actually running unit or build tests here.

Reply:
Sure